.travis.yml

```
@@ -45,7 +45,6 @@ before_cache:
 
 cache:
   directories:
-    - "$HOME/.gradle/caches"
     - "$HOME/.ivy2/cache"
+    - "$HOME/.gradle/caches"
     - "$HOME/.jabba/jdk"
-    - "$HOME/.sbt/boot"
```

@dwijnand, can you clarify why this line was added on a previous PR? Is that something that we should or want to add to template control and let it propagate to all samples?

---

Yes

---

I guess you are saying 'yes' to my second question, correct?
> Is that something that we should or want to add to template control and let it propagate to all samples?

---

Yep!  The whole of `~/.sbt` should be cached in fact.

---

I'll follow this up.